content/news/024/index.md

```diff
@@ -589,6 +589,18 @@ follow the project on [Twitter](https://twitter.com/quilkindev).
 [embark-announce]: https://medium.com/embarkstudios/say-hi-to-quilkin-an-open-source-udp-proxy-88577c795204
 [gcp-announce]: https://cloud.google.com/blog/products/gaming/introducing-quilkin
 
+### [Asset handling in Bevy apps][asset_handling_post]
```

Comment:
For consistency with the rest of the sections, can we change the heading to the name of the library?
```suggestion
### [bevy_asset_loader]
```

Reply:
Sure :+1: